app/controllers/cinema_halls_controller.rb

```diff
@@ -1,4 +1,76 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+	def index 
+		@cinema_halls = Cinema_hall.all
+
+		respond_to do |format|
```

Comment:
You are building API application so you do not have any views provided by Rails. To return data as REST API you can change the action to:
```
def index 
  @cinema_halls = Cinema_hall.all
  render  json: @cinema_halls 
end
```